doc/whatsnew/2/2.14/full.rst

```diff
@@ -90,6 +90,11 @@ Release date: 2022-06-01
 
   Refs #6462
 
+* Fixed a false positive regression in 2.13 for ``used-before-assignment`` where it is safe to rely
```

Comment:
```suggestion
* Fixed a false positive regression in 2.14 for ``used-before-assignment`` where it is safe to rely
```
I think it's a typo in the original issue ? I did not check, but as we just released 2.14...

Reply:
It was caused in 2.13. I figure the reporter was pinning to <1.